search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -65,11 +107,27 @@ function setUserSearchTopic(topic, countries) {
 }
 
 /**
- * Fetches current top trends from the backend and displays them on the website.
+ * Changes `currentTimeRange` parameter and updates trends for new time range.
+ * @param {number} timeRange The interval value for the time range.
  */
-function updateTrends() {
-  fetch('/trends').then(globalTrends => globalTrends.json()).then(trends => {
-    topTrends = trends;
-    setTopTrends(); 
-  });
+function setTimeRange(timeRange) {
+  if (currentTimeRange === timeRange) {
+    return;
+  }
+  currentTimeRange = timeRange;
+  updateTrends();
 }
+
+/**
+ * Fetches current top trends from back end and displays them on the website.
+ * @param {boolean=} setNewTrendEnabled Boolean for whether or not to
+ *     call setNewTrend at the end of setTopTrends.
```

Comment:
You want to write the explanation in words. Not in terms of the functions called.